cmd/director.go

```diff
@@ -63,4 +63,10 @@ func init() {
 	if err != nil {
 		panic(err)
 	}
+
+	directorServeCmd.Flags().BoolP("enable-hostname-redirects", "", false, "Enabling host-aware redirects allows the director to bypass its default response for hosts specified via the Pelican configuration file")
```

Comment:
Let's skip the command line flag if it's not possible to use this functionality completely from CLI.